tests/query/snql/test_invalid_queries.py

```diff
@@ -80,6 +80,16 @@
         "Parsing error on line 1 at '  ORDER BY f '",
         id="aliases are only in the select",
     ),
+    pytest.param(
+        "MATCH (discover_events) SELECT arrayMap((x) -> identity(`x`), sdk_integrations) AS sdks WHERE project_id = 1 AND timestamp >= toDateTime('2021-01-01') AND timestamp < toDateTime('2021-01-02')",
+        "Parsing error on line 1 at 'ap((x) -> ide'",
+        id="identifiers have backticks",
+    ),
+    pytest.param(
+        "MATCH (discover_events) SELECT arrayMap((`x`) -> `x`, sdk_integrations) AS sdks WHERE project_id = 1 AND timestamp >= toDateTime('2021-01-01') AND timestamp < toDateTime('2021-01-02')",
+        "Parsing error on line 1 at 'ap((`x`) -> `'",
```

Comment:
```
arrayMap((`x`) -> `x`
      {          }^
           ^      |
prefix to error   error happens here since the identifier is not a function
```

The parsing error is based on parsimonious (which doesn't have great messages) and is appending an arbitrary prefix from the original string to provide context. The `ap` is the end of `arrayMap`.